src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/breadcrumbs.tsx

```diff
@@ -177,36 +203,54 @@ class BreadcrumbsContainer extends React.Component<Props, State> {
 
   handleFilter = (breadcrumbFilterGroups: BreadcrumbFilterGroups) => () => {
     //types
-    const breadcrumbFilterGroupTypes = breadcrumbFilterGroups.filter(
-      breadcrumbFilterGroup => breadcrumbFilterGroup.groupType === 'type'
-    );
-
-    // TODO(Priscila): implement levels
-    // const breadcrumbFilterGroupLevels = breadcrumbFilterGroups
-    //   .filter(breadcrumbFilterGroup => breadcrumbFilterGroup.groupType === 'level')
-    //   .map(breadcrumbFilterGroup => breadcrumbFilterGroup.type);
+    const breadcrumbFilterGroupTypes = breadcrumbFilterGroups
+      .filter(
+        breadcrumbFilterGroup =>
+          breadcrumbFilterGroup.groupType === 'type' && breadcrumbFilterGroup.isChecked
+      )
+      .map(breadcrumbFilterGroup => breadcrumbFilterGroup.type);
+
+    //levels
+    const breadcrumbFilterGroupLevels = breadcrumbFilterGroups
```

Comment:
Like, the file is already breadcrumbs.tsx, why do all the variables need to be prefixed with `breadcrumb` 😬

Reply:
makes sense..I will rename them in a follow-up PR